modules/update-best-practices.adoc

```diff
@@ -28,6 +28,15 @@ Failing to address critical alerts before beginning an update can cause problema
 
 In the *Administrator* perspective of the web console, navigate to *Observe* -> *Alerting* to find critical alerts.
 
+[id="remove-duplicated-headers-haproxy_{context}"]
+=== Ensure that duplicated encoding headers are removed
+
+Before updating, you will receive a `DuplicateTransferEncodingHeadersDetected` alert if any route records a duplicate `Transfer-Encoding` header issue. This is due to the upgrade from HAProxy 2.2 in previous {product-title} releases to HAProxy 2.6 in {product-title} {product-version}. Failing to address this alert will result in your application pods becoming unreachable if they expose routes using any of these headers.
```

Comment:
To be clear, only the _problematic_ application pods will become unreachable, only through their _routes_, and only if these applications send _multiple_ `transfer-encoding` headers.  
```suggestion
Before updating, you will receive a `DuplicateTransferEncodingHeadersDetected` alert if any route records a duplicate `Transfer-Encoding` header issue. This is due to the upgrade from HAProxy 2.2 in previous {product-title} releases to HAProxy 2.6 in {product-title} {product-version}. Failing to address this alert will result in applications that send multiple `Transfer-Encoding` headers becoming unreachable through routes.
```